index.d.ts

```diff
@@ -57,6 +57,65 @@ export interface constructorOptions {
 export type apiCallback = () => void;
 export type FlushCallback = (error?: Error, data?: any) => void;
 
+export type IdentifyParams = {
+  userId?: string;
+  anonymousId?: string;
```

Comment:
For all the event types, ensure either user ID or anonymous ID is present.

Reply:
Addressed